node-src/lib/log.ts

```diff
@@ -93,25 +115,36 @@ const fileLogger = {
   },
 };
 
-export const createLogger = () => {
-  let level = (LOG_LEVEL.toLowerCase() as keyof typeof LOG_LEVELS) || DEFAULT_LEVEL;
-  if (DISABLE_LOGGING === 'true') level = 'silent';
+/* eslint-disable-next-line complexity */
```

Comment:
Should we refactor this function to lower the complexity instead of disabling the rule?

Reply:
I tried, and it wasn't getting any simpler. Pulling chunks of code out into separate functions only adds extra layers of indirection, it doesn't actually simplify anything.

Reply:
Gotcha. We might have to rethink how logging is structured then. That's completely outside of the scope of this change though so don't worry about that. Thanks for trying!